### ISSUE TITLE
 There is no index for Solar System (jita).

### ISSUE THREAD

**Issue by 4tt1c** [Kyria/LazyBlacksmith#94]
I can't select any other system and there are no prices.

If i select a blueprint i get this error:
There is no index for Solar System (jita).

When i try to switch the system to Amarr, i get the following:
There is no index for Solar System (amarr,jita).

Everything else is working like a charm.
Any advise?

**Comment by 4tt1c** [comment]
After logging in and trying to set a system in the industry settings i get the following message:
Production preferences updated, solarsystem not updated as the system does not exist or does not have any index.

**Comment by Kyria** [comment]
wait, i think I understood your issue, it's for your own instance ?

Did you run celery / celery beat to update the indexes?

**Comment by 4tt1c** [comment]
Yes, it's a self hosted instance.

I've done all the steps from the docs and can access the site as intended.
Only when it comes to price calculations, the website returns the mentioned warnings.

Nginx logs are fine - only that i get a 404 response.

I've also checked the requests the official hosted version is sending and it looks identical. But his version is working fine.
So it might be a problem with either the Nginx config, the gunicorn config some script configuration.

I've also bound the script directly to the static IP of my server to make sure there is no problem in the response chain, but even then the error occurs.

So in short, i've done all the docs steps, i followed your hint from another issue "serve flask app with gunicorn and nginx" from digitalocean.
